src/geometamaker/models.py

```diff
@@ -119,6 +119,67 @@ class TableSchema(Parent):
     See https://datapackage.org/standard/table-schema/#foreignKeys
     """
 
+    def _get_field(self, name):
```

Comment:
These methods were moved from `TableResource` because `VectorResource` no longer subclasses `TableResource`. So `TableSchema` is now a useful place where these methods can be shared by Tables and Vectors.